packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -79,7 +79,7 @@ export type FormConfig<T extends { __typename?: string }> = {
 
 export type TabsConfig = { type: "tabs"; tabs: { name: string; content: GeneratorConfig }[] };
 
-export type DataGridSettings = Pick<GridColDef, "headerName" | "width" | "minWidth" | "maxWidth" | "flex" | "pinned">;
+type DataGridSettings = Pick<GridColDef, "headerName" | "width" | "minWidth" | "maxWidth" | "flex" | "pinned"> & { tooltipMessage?: string };
```

Comment:
Removing this export probably causes the lint error. Also, now that you add custom fields to this, DataGridSettings is not the best name IMO. Maybe BaseColumnConfig instead?

Reply:
@jamesricky what do you think about this rename (I proposed it)?

Reply:
@johnnyomair I like `BaseColumnConfig` 👍🏼